### PR TITLE
issue#1954 Add getWXRenderListener function

### LINKED ISSUE #1954: WXSDKInstance类有必要暴露IWXRenderListener和IWXStatisticsListener两个属性
RT:
  mRenderListener和mStatisticsListener两个属性都没有get方法，如果要对线上项目监听加逻辑需要打开每个类然后添加相同逻辑，比较麻烦并且容易出错
  如果暴露了get方法可以在一个类进行包装，原逻辑不需要修改。
```
WXSDKInstance wxsdkInstance = WXSDKManager.getInstance().getSDKInstance(instanceId)
if (wxsdkInstance != null) {
    // get rander listener
    final IWXRenderListener listener = wxsdkInstance.getRenderListener(); // TODO
    // wrapper listener
    if (listener != null) {
            wxsdkInstance.registerRenderListener(new WXRenderListenerWrapper(listener));
    } else {
            wxsdkInstance.registerRenderListener(new WXRenderListenerWrapper());
    }
}

WXRenderListenerWrapper implements IWXRenderListener {
    @Override public void onException(WXSDKInstance instance, String errCode, String msg) {
        if (listener != null) {
            listener.onException(instance, errCode, msg);
        }
        // do ...
    }
    ....
}
```

### PR DESCRIPTION
**Pull request by wangguanquan**
[android] close #1954 暴露WXSDKInstance#getWXRenderListener方法，为业务提供包装功能

### CODE REVIEW

**Comment by wangguanquan** [comment]
[android] close #1954 Expose the WXSDKInstance#getWXRenderListener method to provide wrapper functionality for the business

File: android/sdk/src/main/java/com/taobao/weex/ui/view/WXTextView.java [hunk 1]
```diff
@@ -86,7 +86,7 @@ public WXGesture getGestureListener() {
 
   @Override
   public CharSequence getText() {
-    return textLayout.getText();
+    return textLayout != null ? textLayout.getText() : null;
```
Comment (YorkShen) [inline android/sdk/src/main/java/com/taobao/weex/ui/view/WXTextView.java]: If you need fix this NPE problem, make another PR, please

**Comment by wangguanquan** [comment]
Fix WXTextView#getText() throws NullPointerException when textLayout not instance

**Comment by YorkShen** [comment]
Move the following change to another PR, please

> Fix WXTextView#getText() throws NullPointerException when textLayout not instance
